### PR TITLE
remove npm badge

### PR DESCRIPTION
**Pull request by fdiblen**
I have removed the npm badge because it is not adding a lot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # the FAIR extension
 
-[![npm version](https://badge.fury.io/js/npm.svg)](https://badge.fury.io/js/npm)
 [![npm version](https://img.shields.io/badge/license-Apache%202-blue)](https://opensource.org/licenses/Apache-2.0)
 [![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow)](https://fair-software.eu)
```
Comment (pedrohserrano) [inline README.md:4]: It would be great if we can have the 5 stars badge

File: README.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 # the FAIR extension
 
-[![npm version](https://badge.fury.io/js/npm.svg)](https://badge.fury.io/js/npm)
 [![npm version](https://img.shields.io/badge/license-Apache%202-blue)](https://opensource.org/licenses/Apache-2.0)
```
Comment (Alialsaeedi1) [inline README.md:3]: I think this should be removed as well.

**Review by Alialsaeedi1 — CHANGES_REQUESTED** [review]

**Review by pedrohserrano — COMMENTED** [review]
Are there any other relevant badges for the repo?

File: README.md (hunk 1 above)
Comment (fdiblen) [reply README.md:4]: Do you have an issue about this? I think this should be part of another PR?

**Review by pedrohserrano — APPROVED** [review]
Yes, it makes sense to remove this badge

File: README.md (hunk 2 above)
Comment (fdiblen) [reply README.md:3]: I think it is a good idea to keep it.

Comment (Alialsaeedi1) [reply README.md:3]: Sure thing!

**Comment by fdiblen** [comment]
> Are there any other relevant badges for the repo?

We should create an issue about this.

**Review by Alialsaeedi1 — APPROVED** [review]
Good job

**Pull request merged**